sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java

```diff
@@ -26,6 +26,7 @@
  * Config center.
  * 
  * @author wangguangyuan
+ * @author dongzonglei 
```

Comment:
When you plan to graduate ,  you will remove all the  author  info of this repo .

Reply:
Thank you for your reminder, I will remove it.

Reply:
We can start a discuss in mailing list before graduate, and remove the author in one pull request.

Reply:
OK.